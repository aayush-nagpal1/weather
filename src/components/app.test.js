import React from 'react';
import App from './app';
import renderer from 'react-test-renderer';

describe("App Component", ()=>{
  it('Application renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})
