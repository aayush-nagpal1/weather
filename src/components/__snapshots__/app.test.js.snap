// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Application renders correctly 1`] = `
<div
  className="App"
>
  <div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12"
        >
          <nav
            className="navbar navbar-expand-lg navbar-light bg-light"
          >
            <a
              className="navbar-brand"
              href="#"
            >
              Weather App
            </a>
            <button
              aria-controls="navbarSupportedContent"
              aria-expanded="false"
              aria-label="Toggle navigation"
              className="navbar-toggler"
              data-target="#navbarSupportedContent"
              data-toggle="collapse"
              type="button"
            >
              <span
                className="navbar-toggler-icon"
              />
            </button>
            <div
              className="collapse navbar-collapse"
              id="navbarSupportedContent"
            />
          </nav>
        </div>
      </div>
    </div>
  </div>
  <div
    className="app-size"
  />
</div>
`;
